Apollo Server with typescript

Sam09040 opened the pull request:
Apollo Server code with schema, resolvers, constructor and instance wrote in Typescript and using tsx hot reload.

Sam09040 committed:
Setting up GraphQL

Setting GraphQL up using a basic schema, also using express server and GraphiQL.

Sam09040 committed:
Setting up GraphQL

Setting GraphQL up using a basic schema, also using express server and GraphiQL.

Sam09040 committed:
Inicial setup

Inicial setup with schema defined.

Sam09040 committed:
Inicial setup

Inicial setup with schema defined.

Sam09040 committed:
Inicial setup

Inicial setup with schema defined.

Sam09040 committed:
Inicial setup

Inicial setup with schema defined.

Sam09040 committed:
Adding main code

Add main files
Add .gitignore file
Delete node_modules directory

Small changes to main and package

Fixing dependencies versions and removing graphql call to console.

Adding prettier and eslint

Adding prettier and eslint modules for code formatting and standardization.

Sam09040 committed:
Add main code

Small changes to main and package

Add:
- main files
- .gitignore file
- prettier and eslint

Delete:
- node_modules directory

Fix:
- dependencies versions and removing graphql call to console.

Sam09040 committed:
Adding .gitignore

Adding .gitignore file and deleting all of the files from the node_modules directory.

Sam09040 committed:
Add main code

Small changes to main and package

Add:
- main files
- .gitignore file
- prettier and eslint

Delete:
- node_modules directory

Fix:
- dependencies versions and removing graphql call to console.

Sam09040 committed:
Working Apollo

Update with the working apollo server, with the defined schema and hot reload.

Add eslint and prettier

Add eslint and prettier
Fix the description
Fix the version of the dependencies.

Small changes

Adding a line at the end of the eslint.config.js and tsconfig.json.

Sam09040 committed:
Working Apollo

Update with the working apollo server, with the defined schema and hot reload.

Sam09040 committed:
Working Apollo

Update with the working apollo server, with the defined schema and hot reload.

Add eslint and prettier

Add eslint and prettier
Fix the description
Fix the version of the dependencies.

Small changes

Add a line at the end of the eslint.config.js and tsconfig.json.

Sam09040 committed:
Working Apollo

Update with the working apollo server, with the defined schema and hot reload.

Sam09040 committed:
Small changes to main and package

Fixing dependencies versions and removing graphql call to console.

racsandoval commented on package.json:
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "botactivities",
+  "version": "1.0.0",
+  "description": "��#\u0000 \u0000b\u0000o\u0000t\u0000A\u0000c\u0000t\u0000i\u0000v\u0000i\u0000t\u0000i\u0000e\u0000s\u0000\r\u0000 \u0000",
```
a descrição ficou estranha

racsandoval commented on dist/index.js:
geralmente não comitamos a pasta `dist`, já que ela é gerada no build do código

Sam09040 committed:
Adding prettier and eslint

Adding prettier and eslint modules for code formatting and standardization.

Sam09040 committed:
Addint eslint and prettier

Adding eslint and prettier. Also fixing the description and fixing the version of the dependencies.

Sam09040 committed:
Add eslint and prettier

Add eslint and prettier
Fix the description
Fix the version of the dependencies.

Small changes

Add a line at the end of the eslint.config.js and tsconfig.json.

racsandoval reviewed (approved):
acho que subiu mais código aqui após pedir a revisão, atenção pra sempre fazer novas funcionalidades em outras branches, para não subir código que ainda não foi revisado

SamuelSsc commented on eslint.config.js:
```diff
@@ -0,0 +1,11 @@
+  {languageOptions: { globals: globals.browser }},
+  pluginJs.configs.recommended,
+  ...tseslint.configs.recommended,
+];
```
nit: adicionar uma linha vazia aqui no fim do arquivo só para não ficar aparecendo esse iconezinho do lint:
<img width="247" alt="image" src="https://github.com/user-attachments/assets/86224e15-e089-4520-9d0a-877ba4ef74bd">

SamuelSsc commented on tsconfig.json:
```diff
@@ -0,0 +1,12 @@
+        "esModuleInterop": true,
+        "types": ["node"]
+    }
+}
```
Add uma linha vazia aqui no fim tbm

Sam09040 committed:
Small changes

Adding a line at the end of the eslint.config.js and tsconfig.json.

Sam09040 committed:
Revert "Adding prettier and eslint"

This reverts commit 52ab9daef64975c569eb3f36fc12a02e450b3419.

Sam09040 committed:
Revert "Small changes to main and package"

This reverts commit cdaa5e99d46ea606a8bed9000a373dcede06944f.

Sam09040 committed:
Revert "Adding .gitignore"

This reverts commit ae18718627f6ccd7701a3920632a59e2e99e5660.

Sam09040 committed:
Revert "Setting up GraphQL"

This reverts commit 5eb19f42294a3d1a3b9cee3ce0c50fb65eb047e8.

Sam09040 committed:
Merge branch 'feature/setting-up-typescript-apollo' of https://github.com/Sam09040/botActivities into origin/develop

Sam09040 committed:
Merge branch 'feature/setting-up-typescript-apollo' of https://github.com/Sam09040/botActivities into feature/setting-up-typescript-apollo
